<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../Desktop/test_desingpqt.qrc">:/newPrefix/Desing.qrc/28.JPEG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnL_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>23</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ligar led1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);
</string>
    </property>
    <property name="text">
     <string>ligar led1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnd1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);
</string>
    </property>
    <property name="text">
     <string>Funçao1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnl2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);
</string>
    </property>
    <property name="text">
     <string>ligar led2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnd2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);</string>
    </property>
    <property name="text">
     <string>Função2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3d">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);
</string>
    </property>
    <property name="text">
     <string>Função3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnl3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 102, 138);</string>
    </property>
    <property name="text">
     <string>ligar led3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../Desktop/test_desingpqt.qrc"/>
  <include location="imagens_pqt.qrc"/>
 </resources>
 <connections/>
</ui>
